e2e/tests/upgrades/genesis_test.go

```diff
@@ -198,12 +197,6 @@ func (s *GenesisTestSuite) HaltChainAndExportGenesis(ctx context.Context, chain
 	state, err := chain.ExportState(ctx, int64(haltHeight))
 	s.Require().NoError(err)
 
-	// state exports currently read from stdout which include log entries, this discards the log entry by the server module
-	// this may need to be updated if log entries are inserted in between exporting of state and the server module logging
-	// see issue: https://github.com/strangelove-ventures/interchaintest/issues/721
-	str := strings.SplitAfter(state, "server")
-	state = str[1]
```

Comment:
The [issue](https://github.com/strangelove-ventures/interchaintest/issues/721) referenced here has been closed.